src/vs/workbench/contrib/terminal/browser/terminalQuickFixBuiltinActions.ts

```diff
@@ -41,8 +41,8 @@ export function gitSimilar(): IInternalOptions {
 				return;
 			}
 			const actions: TerminalQuickFixActionInternal[] = [];
-			const results = matchResult.outputMatch.regexMatch[0].split('\n').map(r => r.trim());
-			for (let i = 1; i < results.length; i++) {
+			const results = matchResult.outputMatch.outputLines.map(r => r.replaceAll('\n', '').trim());
+			for (let i = 3; i < results.length; i++) {
```

Comment:
Instead of hardcoding the line number it starts at, can we scan for the line that the match occurred on (ie. `most similar  ...`) and then add all following lines? I'm concerned about bugs similar to the upstream PR fix where the exact format changes somehow.

Reply:
I thought about that, but it should always be exact since we're counting wrapped lines here.  Also tested in terminals of varying sizes